src/main.rs

```diff
@@ -22,3 +29,16 @@ fn initialize_css() {
         gtk::STYLE_PROVIDER_PRIORITY_APPLICATION,
     );
 }
+
+fn start_gui() {
+    let span = tracing::span!(Level::TRACE, "gui");
+    let _enter = span.enter();
```

Comment:
Normally, it will provide you with the information in which context the log has been printed out. I removed it now, because it doesn't seem to be necessary.